fvcore/nn/jit_handles.py

```diff
@@ -340,6 +378,8 @@ def matmul_flop_jit(inputs: List[Any], outputs: List[Any]) -> typing.Counter[str
     # Inputs contains the shapes of two matrices.
     input_shapes = [get_shape(v) for v in inputs]
     assert len(input_shapes) == 2, input_shapes
+    if len(input_shapes[1]) > 2:
+        return matmul_broadcasted(input_shapes)
     assert len(input_shapes[1]) == 2, input_shapes
     assert input_shapes[0][-1] == input_shapes[1][0], input_shapes
     batch_dim = input_shapes[0][0]
```

Comment:
Instead of creating two new functions, 
```
flop = np.prod(input_shapes[0]) * input_shapes[-1][-1]
```
should be sufficient. Could you implement it this way instead?

Reply:
That's a good idea. Will do

Reply:
I have done it in the simpler way as you suggested now.